### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thehughhefner**
Adding in new information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,29 @@
...
+
+| Title | Status | Released |
+|------|---------|----------|
+[**Do Donpachi**](https://en.wikipedia.org/wiki/DoDonPachi) | Implemented | Yes |
+[**ESP Ra. De.**](https://en.wikipedia.org/wiki/ESP_Ra.De.) | Implemented | Yes |
+[**Dangun Feveron**](https://en.wikipedia.org/wiki/Dangun_Feveron) | **W.I.P** | No |
+[**Air Gallet**](https://en.wikipedia.org/wiki/Air_Gallet) | N/A | No |
+[**Donpachi**](https://en.wikipedia.org/wiki/DonPachi) |  N/A | No |
+[**Gaia Crusaiders**](http://adb.arcadeitalia.net/dettaglio_mame.php?game_name=gaia&arcade_only=0&autosearch=1) | N/A | No |
+[**Gunwange**](https://en.wikipedia.org/wiki/Guwange) | N/A | No |
```
Comment (asturur) [inline README.md:17]: pretty much sure there was a beta of this?

Comment (thehughhefner) [reply README.md:17]: If those are in beta, then I can update the status for those games. Maybe @nullobject can confirm

**Comment by asturur** [comment]
gunwange and uon poko are in patreon beta i can confirm, i just checked the
old patreon messages.

**Comment by thehughhefner** [comment]
I updated the table now

**Review by nullobject — APPROVED** [review]
Thanks folks, appreciate this nice little table :sparkling_heart:

**Pull request merged**